source/plugins/cloudstack-kubernetes-service.rst

```diff
@@ -185,22 +187,24 @@ New Kubernetes clusters can be created using the API or via the UI. User will be
 createKubernetesCluster API can be used to create new Kubernetes cluster. It takes following parameters as input,
 
 - **name** (name for the Kubernetes cluster; Required)
-- **description** (description for the Kubernetes cluster; Required)
+- **description** (description for the Kubernetes cluster)
 - **zoneid** (availability zone in which Kubernetes cluster to be launched; Required)
-- **kubernetesversionid** (Kubernetes version with which cluster to be launched; Required)
-- **serviceofferingid** (the ID of the service offering for the virtual machines in the cluster; Required)
+- **kubernetesversionid** (Kubernetes version with which cluster to be launched; Required for CloudManaged clusters)
```

Comment:
is "CloudManaged" the right term? we could be looking at other SDI managed clusters as well and these would not be CloudManaged from our perspective. I think "CloudStack managed" would be a better term.

Reply:
addendum I see that the term is explained further on. I think it should be explained at first use. (not sure if we have a glosary this would fit into)

Reply:
I have moved `clustertype` parameter before `kubernetesversionid`. Is that okay?